Library/Formula/emp.rb

```diff
@@ -0,0 +1,27 @@
+require "language/go"
+
+class Emp < Formula
+  desc "CLI for Empire."
+  homepage "https://github.com/remind101/emp"
+  url "https://github.com/remind101/emp/archive/v0.9.1.tar.gz"
+  version "0.9.1"
+  sha256 "9619ffd25856988d92d39b31d9e7ed6edacbd85543ff6e60300954df7c16135d"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    emppath = buildpath/"src/github.com/remind101/emp"
+    emppath.install Dir["{*,.git}"]
+
+    cd emppath do
+      system "go", "build", "-o", bin/"emp"
+    end
+  end
+
+  test do
+    assert_equal "#{version}",
```

Comment:
It’s ok to fetch small resources for testing, see e.g. `wiki.rb`, which we’ll merge very soon: https://github.com/Homebrew/homebrew/pull/44844/files#diff-b1eede01b0006877d01481b90c0e93e1R29